src/Microsoft.AspNet.Mvc.DataAnnotations/DataAnnotationsModelValidator.cs

```diff
@@ -71,5 +89,29 @@ public IEnumerable<ModelValidationResult> Validate(ModelValidationContext valida
 
             return Enumerable.Empty<ModelValidationResult>();
         }
+
+        private static IDictionary<Type, ValidationAttributeAdapterFactory> AttributeAdapters =
+            ValidationAttributeAdapterTable.AttributeFactories;
+
+        private string GetErrorMessage(ModelMetadata metadata, IModelMetadataProvider metadataProvider)
+        {
+            if (Attribute == null)
+            {
+                throw new ArgumentNullException(nameof(Attribute));
+            }
```

Comment:
`Attribute` isn't an argument and it's already `null` checked in the constructor. Think you meant to check the actual arguments.